### PR TITLE
Rebrand to CourseUp with minor visual tweaks

### PR DESCRIPTION
**Pull request by aomi**
## Overview

Closes #XX
Rebrands the app to CourseUp and changes the color scheme a bit to match the landing image. Adds a :wave: emoji too :+1: 

image

### CODE REVIEW

File: src/app/header/Header.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { Center, Grid, GridItem, Image, Button, Flex } from '@chakra-ui/react';
+import { Center, Grid, GridItem, Image, Button, Flex, Text, LinkBox } from '@chakra-ui/react';
```
Comment (keithradford) [inline src/app/header/Header.tsx:1]: `Image` and `Button` aren't used

File: src/app/landing/index.tsx [hunk 2]
```diff
@@ -1,7 +1,21 @@
 import { Image } from '@chakra-ui/image';
 import { Flex, Grid, GridItem, Heading, Text } from '@chakra-ui/layout';
+import { useEffect, useState } from 'react';
 
 export default function Landing() {
+  const [handIndex, setHandIndex] = useState(0);
```
Comment (keithradford) [inline src/app/landing/index.tsx:6]: emoji code, nice

**Review by keithradford — APPROVED** [review]
LGTM, just a linting thing

File: src/app/header/Header.tsx (hunk 1 above)
Comment (aomi) [reply src/app/header/Header.tsx:1]: Maybe we should finally setup pre-commit hooks :eyes:

Comment (keithradford) [reply src/app/header/Header.tsx:1]: yeah, and make the import linting an error instead of a warning, it's def the most common lol

**Pull request merged**